extract/extract.go

```diff
@@ -423,3 +448,5 @@ func buildTags() (string, error) {
 
 	return currentGoVersion + ",!" + nextGoVersion, nil
 }
+
+func isInStdlib(path string) bool { return !strings.Contains(path, ".") }
```

Comment:
I don't understand this check. Why would there always be a dot in a non-stdlib import path?

Reply:
Any non-stdlib package name would contain a domain name in it, thus with a mandatory `.`.